src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * The main class to deal with interactions with the user.
+ */
+
+public class Ui {
+
+    public static void intro() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hi! I'm duke.Duke your friendly neighbourhood chat bot");
+        System.out.println("What can i do for you?");
+    }
+
+    public static void tasks() {
```

Comment:
Naming of method 'tasks' can be misleading. Perhaps it should 'explain the named entity to the reader accurately and at a sufficient level of detail.'?